lib/packages/core/lib/src/widgets/chatroom/chatroom_menu.dart

```diff
@@ -45,17 +45,23 @@ class _ChatroomMenuState extends State<LMChatroomMenu> {
   ValueNotifier<bool> rebuildChatroomMenu = ValueNotifier(false);
 
   LMChatHomeFeedBloc? homeBloc;
```

Comment:
We can directly initialise homeBloc and other bloc here, i think we don’t have to assign it in initState and didUpdateWidget

Reply:
We are anyway assigning actions in these functions, do you think it will be better performance wise if we make them final?

Reply:
I do not have the complete knowledge on making any variable final in term of performance yet. but i always prefer to make it final whenever possible.

Reply:
Will change this